appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -3964,6 +3964,10 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @Description("")
   String GotTextEvents();
 
+  @DefaultMessage("AfterText")
```

Comment:
It would be better to name this event AfterTextSaved and make the corresponding changes throughout this file and the others.